kroxylicious-api/src/main/java/io/kroxylicious/proxy/filter/RequestFilterResultBuilder.java

```diff
@@ -44,4 +45,16 @@ public interface RequestFilterResultBuilder extends FilterResultBuilder<RequestH
      */
     CloseOrTerminalStage<RequestFilterResult> shortCircuitResponse(@NonNull ApiMessage message) throws IllegalArgumentException;
 
+    /**
+     * Generate a short-circuit error response towards the client.
```

Comment:
We should give a bit more detail on what an 'error response' _is_. Something like this:

```suggestion
     * Generate a short-circuit error response towards the client.
     * Error responses are API-specific, but will use an error code, and possibly error message, appropriate for the given exception type at either at the top level of the response (if the API for the response has a global error code), or for all entities given in the request (if the API for the response has only-per entity error codes).
```

Reply:
reworded slightly in e30d7568bc9cf4fa2409c64e8471173fce2a1913